.github/workflows/test.yml

```diff
@@ -45,4 +49,4 @@ jobs:
           poetry run netexec mssql 127.0.0.1
           poetry run netexec ssh 127.0.0.1
           poetry run netexec ftp 127.0.0.1
-          poetry run netexec smb 127.0.0.1 -M veeam
\ No newline at end of file
+          poetry run netexec smb 127.0.0.1 -M veeam
```

Comment:
Imo we should also add the same commands for `netexec smb ...` as for poetry, so we also check the pipx installation properly. @mpgn @Marshall-Hallenbeck thoughts?